partners/README.md

```diff
@@ -0,0 +1,48 @@
+## Partner tekton task submission process
+
+Partners can submit their tekton task in [build-definitions](https://github.com/redhat-appstudio/build-definitions) repository by sending a PR.
+
+Build-definitions repository CI job will validate the PR against a [set of checks](#checks).
+
+Build-definitions repository maintainers needs to approve and merge the PR before the task can be used.
+
+### How to submit a task
+
+Create your task yaml and put it inside a directory of structure `partners/<task_name>/<task_version>/<task_name>.yaml` and 
+also include a `OWNERS` file inside the task directory `partners/<task_name>/OWNERS`
+
+For example, if your task name is `my_task` and version of the your task is `0.1`, your task yaml and OWNERS file location will be `partners/my_task/0.1/my_task.yaml` and `partners/my_task/OWNERS`
+
+Send a PR to the [build-definitions](https://github.com/redhat-appstudio/build-definitions) repository containing your task yaml and OWNERS file.
+
+### Checks
+Task is validated against the following mandatory checks:
+* Task follows the correct directory structure and includes OWNERS file (This check is a prerequisites for other checks to be continued)
+* Task is not using any privilege escalations
+* Task schema is a valid one i.e. the task resource can be applied in the openshift cluster
```

Comment:
I would think about changing `Task` to `The task`

Reply:
I would prefer to keep `Task` here.